fem/linearform.cpp

```diff
@@ -169,7 +169,7 @@ void LinearForm::Assemble()
          MFEM_ASSERT(bdr_marker.Size() == bdr_attr_marker.Size(),
                      "invalid boundary marker for boundary integrator #"
                      << k << ", counting from zero");
-         for (int i = 0; i < bdr_attr_marker.Size(); i++)
+         for (i = 0; i < bdr_attr_marker.Size(); i++)
```

Comment:
I think it's better to just move `int i` into the other for loops, `i` doesn't need to exist in a broader scope

Reply:
Fixed